Marlin/ultralcd.cpp

```diff
@@ -3460,17 +3457,15 @@ void lcd_update() {
   #if ENABLED(ULTIPANEL)
     static millis_t return_to_status_ms = 0;
     manage_manual_move();
-  #endif
 
-  lcd_buttons_update();
+    lcd_buttons_update();
```

Comment:
I tested it with my ADC_KEYPAD at at first glance it still works with this change. I did not try any UBL stuff though.